### PR TITLE
Fix startup check for JSON logging

### PR DESCRIPTION
**Pull request by paolobueno**
# Motivation

Change the recipe to check for logs in the new format when confirming EAP service startup.
# Changes
- update grep regex to match new logger output

ping @matzew @mikenairn

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 license 'All rights reserved'
 description 'Installs/Configures jboss7'
 long_description 'Installs/Configures jboss7'
-version '0.11.2'
+version '0.11.3'
```
Comment (mikenairn) [inline metadata.rb:7]: Needs to be 0.12.0 going back to master.

**Review by mikenairn — COMMENTED** [review]
Have these changes been tested and verified on both ubuntu and rhel?

**Comment by paolobueno** [comment]
@mikenairn actually I was just about to ask you if there was a way I could test this on RHEL through one of the CI builds you pointed out earlier. ATM I can only test on my ubuntu single dev vm/cluster since I don't have a RHEL deployment handy.

As a side note should these changes rather be on the wrapper cookbook in fhcap instead? We can close this and I can make the changes there instead.

**Comment by mikenairn** [comment]
@paolobueno You know what, I reviewed that thinking it was the wrapper cookbook :-)

Yes this is the wrong place to do this, these changes should be made in the init scripts in the wrapper.

**Pull request closed without merging**